New snippets

DominicVonk opened the pull request.

DominicVonk committed:
Merge remote-tracking branch 'origin/patch-2'

DominicVonk committed:
Merge remote-tracking branch 'origin/patch-3'

DominicVonk committed:
:sparkles: added some new snippets also in docs :memo:

DominicVonk committed:
Merge branch 'master' into master

selvinortiz commented:
@DominicVonk Nice work👍 

If you don't mind, I'd like to run a couple of things by you before I approve/reject the PR.

1. I'm on the fence about using `repl` instead of `replace` as a _prefix_.
2. Whether we use `repl` or `replace`, I'd like to have the _regx_ version to be a suffix rather than prefix. For example: `replreg` or `replacereg`
3. Related to (2). Would you consider using an `x` to denote the _regx_ version of `repl|replace`? For example: `replx` or `replacex`.
4. I love `random` and `split`, those are great👍
5. I really don't know if I want to add `loginform`, just because people always seem to find ways to complain about the generated HTML. If we did use `loginform`, we may want to implement the other example templates (user registration form, search form, set password form, etc) in the docs.

Let me know what you think.

DominicVonk commented:
On points: 1,2,3; I think that's good (I will rename them to replace and replacex).
On point: 5; I think we could also implement the other example templates, because it's a snippet plugin. We should use the cleanest versions of it.

DominicVonk committed:
:sparkles: added new snippets + in docs :memo:
Added:
+ userprofileform
+ userregistrationform
+ forgotpasswordform
+ setpasswordform
+ searchform
+ searchformr
+ rss

Changed:
* loginform

DominicVonk committed:
Merge branch 'master' of github.com:DominicVonk/twig-for-craft-2
